.github/workflows/bump-versions.yaml

```diff
@@ -30,15 +30,51 @@ on:
 jobs:
   #label PR as being compatible with the `update-versions` command
   labelPR:
-    name: Label PR for update-versions
+    name: Set merge-and-release-extension-compatible label
```

Comment:
I changed the name since I think the new name would be more intuitive for a random engineer. Not deeply attached to this change.